sources/healthdog/src/main_test.rs

```diff
@@ -0,0 +1,208 @@
+use crate::error::{Error, Result};
+use crate::main_inner;
+use crate::service_check::{ServiceCheck, ServiceHealth};
+use httptest::responders::status_code;
+use httptest::{matchers::*, Expectation, Server};
+use std::fs::write;
+use std::path::PathBuf;
+use tempfile::TempDir;
+
+const OS_RELEASE: &str = r#"PRETTY_NAME=Bottlerocket,
```

Comment:
```suggestion
const OS_RELEASE: &str = r#"PRETTY_NAME=Bottlerocket
```
Guessing that was a typo?

Reply:
Fixed in [2a5aef0](https://github.com/bottlerocket-os/bottlerocket/compare/f515d383c9e181c8a531a02414b320ff9231a81f..2a5aef04d7abcae5c91bf8cec84b8e95c7ccd306)